1.0/nanoserver/sdk/Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM microsoft/dotnet:1.0.4-sdk-nanoserver-10.0.14393.953
 # Install node
 ENV NODE_VERSION 6.10.2
 
-RUN Invoke-WebRequest https://nodejs.org/dist/v${env:NODE_VERSION}/node-v${env:NODE_VERSION}-win-x64.zip -outfile node.zip; `
+RUN Invoke-WebRequest -UseBasicParsing https://nodejs.org/dist/v${env:NODE_VERSION}/node-v${env:NODE_VERSION}-win-x64.zip -outfile node.zip; `
```

Comment:
> make this the default behavior .... different behavior across OSes

+1 like 1000x. One of the main points of Docker is to provide consistent environment for apps